<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureNetworkInterfaceDialog</class>
 <widget class="QDialog" name="ConfigureNetworkInterfaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure network interface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="cboModel"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblMac">
     <property name="text">
      <string>MAC:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="txtMacAddress">
     <property name="inputMethodHints">
      <set>Qt::ImhUppercaseOnly</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>175</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblModel">
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="txtName"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>cboModel</tabstop>
  <tabstop>txtMacAddress</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureNetworkInterfaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureNetworkInterfaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
